bin/rose-make-docs

```diff
@@ -40,6 +40,7 @@
 #     --default-version=VERSION
 #         By default the current version is symlinked as the default version,
 #         provide an alternative version to override this.
+#         If set to 'none' then the default version will remain unchanged.
```

Comment:
Throughout ``travis-scripts`` ``--default-version none`` is used as opposed to ``--default-version=none`` as documented. I've tested these out & only the former syntax seems to work, so ideally we can update this docstring accordingly to prevent confusion.

Reply:
Well spotted!